core/views/browse.py

```diff
@@ -504,7 +506,7 @@ def page_ocr_xml(request, lccn, date, edition, sequence):
 def page_ocr_txt(request, lccn, date, edition, sequence):
     title, issue, page = _get_tip(lccn, date, edition, sequence)
     try:
-        text = page.ocr.text
+        text = get_page_text(page)
         return HttpResponse(text, content_type='text/plain')
```

Comment:
As a style point, I'd prefer to move the return below the try block since that exception won't be thrown by the second statement.

Reply:
moved below